lib/cassia/response_handlers/open_notify.rb

```diff
@@ -18,15 +18,14 @@ def handle(response)
       private
 
       def handle_success
-        @aps.each do |router|
-          router.notification_on = true
+        routers_to_be_turned_on = @access_controller.routers.select {|router| @aps.include?(router.mac) }
```

Comment:
is it possible for the user to pass the `*`? If so then we should update all routers